### PR TITLE
Changes to college and some to admin panel

### PR DESCRIPTION
**Pull request by RobertPlaiasu**

### CODE REVIEW

File: Stoodle/app/Http/Controllers/CollegeController.php [hunk 1]
```diff
@@ -224,14 +167,31 @@ public function update(Request $request, $id)
...
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy( $id )
+    {
+        $college = College::findOrFail( $id );
+        $college->delete();
+        redirect('/facultati');
+    }
```
Comment (GrigoAleex) [inline Stoodle/app/Http/Controllers/CollegeController.php:186]: Couldn't you also change the param form `id` to `College college`?

File: Stoodle/resources/views/admin/edit.blade.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
 @section('title', 'Creare profil')
 
 @section('content')
-    <h1> Adaugare {{ $text }} in baza de date</h1>
+    <h1> Schimba {{ $text }} in baza de date</h1>
```
Comment (GrigoAleex) [inline Stoodle/resources/views/admin/edit.blade.php:6]: I don't think that **Schimba** is a better word in this context. What do you think?

File: Stoodle/app/Http/Controllers/CollegeController.php (hunk 1 above)
Comment (RobertPlaiasu) [reply Stoodle/app/Http/Controllers/CollegeController.php:186]: yes you are right i forgot about that

File: Stoodle/resources/views/admin/edit.blade.php (hunk 2 above)
Comment (RobertPlaiasu) [reply Stoodle/resources/views/admin/edit.blade.php:6]: This is for edit and you don't add you change

**Pull request closed without merging**